### PR TITLE
Fix NL Objective Quadratic Indexing

### PR DESCRIPTION
**Pull request by ccoffrin**
@frederikgeth 🤣

Mine wins because I added a note to the change log as well.

### CODE REVIEW

File: src/core/objective.jl [hunk 1]
```diff
@@ -272,7 +272,7 @@ function _objective_min_fuel_and_flow_cost_polynomial_nl(pm::AbstractPowerModel)
                 gen_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*pg + cost_rev[3]*pg^2)
             elseif length(cost_rev) >= 4
                 cost_rev_nl = cost_rev[4:end]
-                gen_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*pg + cost_rev[2]*pg^2 + sum( v*pg^(d+2) for (d,v) in enumerate(cost_rev_nl)) )
+                gen_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*pg + cost_rev[3]*pg^2 + sum( v*pg^(d+2) for (d,v) in enumerate(cost_rev_nl)) )
```
Comment (frederikgeth) [inline src/core/objective.jl]: doesn't it need to be `v*pg^(d+3)`?

File: src/core/objective.jl [hunk 2]
```diff
@@ -292,7 +292,7 @@ function _objective_min_fuel_and_flow_cost_polynomial_nl(pm::AbstractPowerModel)
                 dcline_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*p_dc + cost_rev[3]*p_dc^2)
             elseif length(cost_rev) >= 4
                 cost_rev_nl = cost_rev[4:end]
-                dcline_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*p_dc + cost_rev[2]*p_dc^2 + sum( v*p_dc^(d+2) for (d,v) in enumerate(cost_rev_nl)) )
+                dcline_cost[(n,i)] = JuMP.@NLexpression(pm.model, cost_rev[1] + cost_rev[2]*p_dc + cost_rev[3]*p_dc^2 + sum( v*p_dc^(d+2) for (d,v) in enumerate(cost_rev_nl)) )
```
Comment (frederikgeth) [inline src/core/objective.jl]: same here

File: src/core/objective.jl (hunk 1 above)
Comment (ccoffrin) [reply src/core/objective.jl]: You are so right.  I'm on it.

**Comment by frederikgeth** [comment]
ready to merge IMO

**Pull request merged**